src/model.ts

```diff
@@ -70,6 +70,12 @@ export default class Model {
     }
   }
 
+  static getFetchOptions() : FetchOptions {
```

Comment:
I like the concept but we might want a different name. This isn't returning the `options` passed to `fetch` - those are generated elsewhere and merged in. If you put `headers` here we would override it. Maybe this is more accurately `enhancedFetchOptions` (probably better name but you see what I'm saying?)